test/e2e/horizontal_pod_autoscaling.go

```diff
@@ -90,32 +90,113 @@ var _ = Describe("Horizontal pod autoscaling", func() {
 	})
 
 	It("[Skipped][Horizontal pod autoscaling Suite] should scale from 5 pods to 3 pods and from 3 to 1 (scale resource: CPU)", func() {
-		rc = NewResourceConsumer("rc", 5, 700, f)
-		createHorizontalPodAutoscaler(rc, "0.3")
+		rc = NewResourceConsumer("rc", 5, 700, 0, f)
+		createCPUHorizontalPodAutoscaler(rc, "0.3")
 		rc.WaitForReplicas(3)
 		rc.ConsumeCPU(100)
 		rc.WaitForReplicas(1)
 		rc.CleanUp()
 	})
 
+	// Memory tests
+	It("[Skipped][Horizontal pod autoscaling Suite] should scale from 1 pod to 3 pods (scale resource: Memory)", func() {
+		rc = NewResourceConsumer("rc", 1, 0, 800, f)
+		createMemoryHorizontalPodAutoscaler(rc, "300")
+		rc.WaitForReplicas(3)
+		rc.CleanUp()
+	})
+
+	It("[Skipped][Horizontal pod autoscaling Suite] should scale from 3 pods to 1 pod (scale resource: Memory)", func() {
+		rc = NewResourceConsumer("rc", 3, 0, 0, f)
+		createMemoryHorizontalPodAutoscaler(rc, "700")
+		rc.WaitForReplicas(1)
+		rc.CleanUp()
+	})
+
+	It("[Skipped][Horizontal pod autoscaling Suite] should scale from 1 pod to maximum 5 pods (scale resource: Memory)", func() {
+		rc = NewResourceConsumer("rc", 1, 0, 700, f)
+		createMemoryHorizontalPodAutoscaler(rc, "100")
+		rc.WaitForReplicas(5)
+		rc.CleanUp()
+	})
+
+	It("[Skipped][Horizontal pod autoscaling Suite] should scale from 1 pod to 3 pods and from 3 to 1 (scale resource: Memory)", func() {
+		rc = NewResourceConsumer("rc", 1, 0, 700, f)
+		createMemoryHorizontalPodAutoscaler(rc, "300")
+		rc.WaitForReplicas(3)
+		rc.ConsumeMem(100)
+		rc.WaitForReplicas(1)
+		rc.CleanUp()
+	})
+
+	It("[Skipped][Horizontal pod autoscaling Suite] should scale from 1 pod to 3 pods and from 3 to 5 (scale resource: Memory)", func() {
+		rc = NewResourceConsumer("rc", 1, 0, 500, f)
+		createMemoryHorizontalPodAutoscaler(rc, "200")
+		rc.WaitForReplicas(3)
+		rc.ConsumeMem(1000)
+		rc.WaitForReplicas(5)
+		rc.CleanUp()
+	})
+
+	It("[Skipped][Horizontal pod autoscaling Suite] should scale from 3 pods to 1 pod and from 1 to 3 (scale resource: Memory)", func() {
+		rc = NewResourceConsumer("rc", 3, 0, 0, f)
+		createMemoryHorizontalPodAutoscaler(rc, "300")
+		rc.WaitForReplicas(1)
+		rc.ConsumeMem(700)
+		rc.WaitForReplicas(3)
+		rc.CleanUp()
+	})
+
+	It("[Skipped][Horizontal pod autoscaling Suite] should scale from 5 pods to 3 pods and from 3 to 1 (scale resource: Memory)", func() {
+		rc = NewResourceConsumer("rc", 5, 0, 700, f)
+		createMemoryHorizontalPodAutoscaler(rc, "300")
+		rc.WaitForReplicas(3)
+		rc.ConsumeMem(100)
+		rc.WaitForReplicas(1)
+		rc.CleanUp()
+	})
+
 })
 
-func createHorizontalPodAutoscaler(rc *ResourceConsumer, cpu string) {
-	hpa := &experimental.HorizontalPodAutoscaler{
+func createCPUHorizontalPodAutoscaler(rc *ResourceConsumer, cpu string) {
```

Comment:
We don't support it in HorizontalPodAutoscaler yet. I'm thinking about adding this, but probably not for 1.1.